ymer/src/main/java/com/avanza/ymer/support/JavaInstantWriteConverter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer.support;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+import org.springframework.core.convert.converter.Converter;
+
+public class JavaInstantWriteConverter implements Converter<Instant, String> {
+    @Override
+    public String convert(Instant instant) {
+        return instant.atZone(ZoneId.systemDefault())
```

Comment:
In the read-converter, we do not use any timezone, but instead use `Instant.parse` (which internally calls `DateTimeFormatter.ISO_INSTANT.parse(..)`). To be consistent, maybe we should use `Instant::toString()` here, or `DateTimeFormatter.ISO_INSTANT.format(..)`, so that it is obvious that the system's timezone is not used.

Reply:
Good idea! new commit after code review suggestion is now available @oskarwiksten